cdap-ui/app/cdap/components/PipelineSummary/index.js

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, {Component, PropTypes} from 'react';
+import {fetchSummary} from 'components/PipelineSummary/PipelineSummaryActions';
+import PipelineSummaryStore from 'components/PipelineSummary/PipelineSummaryStore';
+import {convertProgramToApi} from 'services/program-api-converter';
+import RunsHistoryGraph from 'components/PipelineSummary/RunsHistoryGraph';
+import LogsMetricsGraph from 'components/PipelineSummary/LogsMetricsGraph';
+// import NodesMetricsGraph from 'components/PipelineSummary/NodesMetricsGraph';
+import { DropdownToggle, DropdownItem } from 'reactstrap';
+import CustomDropdownMenu from 'components/CustomDropdownMenu';
+import {UncontrolledDropdown} from 'components/UncontrolledComponents';
+import IconSVG from 'components/IconSVG';
+import T from 'i18n-react';
+
+const PREFIX = 'features.PipelineSummary';
+
+require('./PipelineSummary.scss');
+const ONE_DAY_SECONDS = 86400;
+
+export default class PipelineSummary extends Component {
+  constructor(props) {
+    super(props);
+    let {namespaceId, appId, programType, programId, pipelineConfig} = props;
+    this.state = {
+      runs: [],
+      logsMetrics: [],
+      nodesMetrics: [],
+      totalRunsCount: 0,
+      runsLimit: 10,
+      filterType: 'limit',
+      start: 0,
+      activeRunsFilter: 'Last 10 runs',
```

Comment:
Can use `T.translate(`${RUNSFILTERPREFIX}.last10Runs` instead of hardcoding I guess